static/app/utils/analytics/feedbackAnalyticsEvents.tsx

```diff
@@ -18,6 +22,7 @@ export type FeedbackEventParameters = {
 export type FeedbackEventKey = keyof FeedbackEventParameters;
 
 export const feedbackEventMap: Record<FeedbackEventKey, string | null> = {
+  'feedback.anomaly-detection-feedback-submitted': 'Anomaly Detection Feedback Submitted',
```

Comment:
While this _is_ feedback, the events defined here are related to the Feedback product. So I don't think this is the right place. I don't know if alerts has its own file or not?

Reply:
Okay, I see. Alerts doesn't have its own file, so I can make one.